riak_mesos/commands/cmd_framework.py

```diff
@@ -76,20 +76,18 @@ def wait_for_service(ctx, **kwargs):
     """Waits timeout seconds (default is 60) or until Framework is running.
     Specify timeout with --timeout."""
     ctx.init_args(**kwargs)
-
-    def inner_wait_for_framework(seconds):
-        if seconds == 0:
+    timeout = ctx.timeout
+    while timeout >= 0:
+        if timeout == 0:
```

Comment:
We should either log about not responding after making the last request, or skip the last request when `timeout == 0`. As this stands, it'll log to say it didn't respond then try one more time :)